docs/topic_guides/theory/linked_data_as_graphs.md

```diff
@@ -20,4 +20,26 @@ When linking people together, a cluster represents the all of the records in our
 
     For clusters produced with Splink, every edge comes with an associated Splink score (the probability of two records being a match). The clustering threshold (`match_probability_threshold`) supplied by the user determines which records are included in a cluster, as any links (edges) between records with a match probability below this threshold are excluded.
 
-    Clusters, specifically cluster IDs, are the ultimate output of a Splink pipeline.
\ No newline at end of file
+    Clusters, specifically cluster IDs, are the ultimate output of a Splink pipeline.
+
+## Probabilistic data linkage and graphs
+
+When using [probabilistic linkage](./probabilistic_vs_deterministic.md), each pair of records has a score indicating how similar they are. For example, consider a collection of records with pairwise similarity score:
+
+![Threshold Cluster](../../img/clusters/threshold_cluster.drawio.png){:width="80%"}
+
+Having a score associated with each pair of records is the key benefit of probabilistic linkage, as we have a measure of how similarity of the records (rather than a binary link/no-link). However, we need to choose a threshold above which links are considered valid in order to generate oue final linked data (clusters).
+
```

Comment:
```suggestion
Having a score associated with each pair of records is the key benefit of probabilistic linkage, as we have a measure of similarity of the records (rather than a binary link/no-link). However, we need to choose a threshold above which links are considered valid in order to generate our final linked data (clusters).

```